### DISCUSSION TITLE
Are there any ways to send a webhook -> outbound

### DISCUSSION THREAD

**Discussion by luis-n4489** [concourse/concourse#6547] [Help & Support] [answered]
Since I am trying to interact with an Event Mgmt system (Stackstorm in this case) i wanted to see if it was possible to send an outbound webhook with a json payload upon success or failure etc. 

I might have a work around, send a slack msg to a chatbot connecting to stackstorm and then execute a workflow but that wouldnt really be the best way.

**Comment by taylorsilva** [comment, marked as answer]
Have you looked at using the `on_*` steps? https://concourse-ci.org/jobs.html#schema.job.on_success
You can use those to run a task/resource that does whatever you want.

When our pipelines fail we have a slack notification sent to us using the [slack-notifier resource](https://github.com/mockersf/concourse-slack-notifier): https://github.com/concourse/ci/blob/8868b05b124e1b2389e2b06bcd0b8ef2a73c2bc9/pipelines/concourse.yml#L89-L93

**Reply by luis-n4489** [reply]
RIght but those are for resources that were created to do so, i guess what my question is.. is there a resource out there to do so? Or will i have to create one?

I get how to use on_* etc, and i could possibly create yml script / task to run once successful but was wondering if something like this was created.

**Reply by taylorsilva** [reply]
I don't feel like I'm 100% understanding you, so sorry if the following misses the mark.

Are you looking for a stackstorm resource? Or just a general tool for sending webhooks?
If the former, I'm guessing there is no stackstorm concourse resource :(
If it's the latter, I'm not aware of a nice tool for that either. I'd probably write a quick bash script (or any other scripting language works too) to write what I need.

**Reply by luis-n4489** [reply]
The former, i need to be able to send a webhook out from concourse to any other app with a json payload if possible.

**Reply by taylorsilva** [reply]
Would this work for you? Haven't used it myself, just searched and found this: https://github.com/mdb/concourse-webhook-resource
or even this? https://github.com/vmware-tanzu-labs/concourse-curl-resource

**Reply by luis-n4489** [reply]
yes! wow, i can't believe i did not find those and i looked for a bit. thank you @taylorsilva

**Reply by taylorsilva** [reply]
No problem! If it helps, my search term was "concourse webhook resource"
and I'm using DuckDuckGo as my search engine :)
